models/CoronSegmentator/configs/metadata.json

```diff
@@ -0,0 +1,88 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

Comment:
Hi @Foxconn-bgroup-aidd, could you please utilize the latest schema available at the following link:
"https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json"

Based on the content, it appears that you are already using the most recent version, as it includes "required_packages_version." Thanks.